python-sdk/.test_durations

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
@kaxil the philosophy from the pytest-split people is that small changes in durations doesn't really matter and you can always rerun later if you add a LOT of tests. So if you add more tests it will just assume each test is about the average runtime. I think we can later on figure out a way of automating it, but for the sake of short-term gains I think we should consider this a "perfect enemy of the good" type of situation.

Reply:
> I think we can later on figure out a way of automating it, but for the sake of short-term gains I think we should consider this a "perfect enemy of the good" type of situation.

I don't have problem with "perfect enemy of the good" type of situation but most often than not, it translates into "it will never change" situation :) . Atleast do create a GitHub issue so that you or someone can take care of it in future.

One possible solution would be something like  https://github.com/mlfoundations/open_clip/blob/9b0351a9de39debec58df3ff9b90176bd9f8b01e/.github/workflows/ci.yml#L60-L65

Reply:
Oh neat! Let me play with that for an hour or two tomorrow and see if I can get it to work.